cmd/medius/common/registry.go

```diff
@@ -25,43 +25,50 @@ type Entry struct {
 var staticRegistry = []Entry{
 	{
 		Artifacts: []api.Artifact{
-			centos.New("8.4", nil),
+			centos.New("8.4", "x86_64", nil),
+			centos.New("8.4", "aarch64", nil),
 		},
 		UseForDocs: false,
 	},
 	{
 		Artifacts: []api.Artifact{
-			centos.New("7-2009", defaultEnvVariables("u1.small", "centos.7")),
+			centos.New("7-2009", "x86_64", defaultEnvVariables("u1.small", "centos.7")),
+			centos.New("7-2009", "aarch64", defaultEnvVariables("u1.small", "centos.7")),
 		},
 		UseForDocs: true,
 	},
 	{
 		Artifacts: []api.Artifact{
-			centosstream.New("9", &docs.UserData{Username: "cloud-user"}, defaultEnvVariables("u1.small", "centos.stream9")),
+			centosstream.New("9", "x86_64", &docs.UserData{Username: "cloud-user"}, defaultEnvVariables("u1.small", "centos.stream9")),
+			centosstream.New("9", "aarch64", &docs.UserData{Username: "cloud-user"}, defaultEnvVariables("u1.small", "centos.stream9")),
```

Comment:
IIRC we need different instancetypes and preferences for ARM guests.

@lyarwood I think it is better to leave env variables for arm64 this out at the moment, right?

Reply:
Yes drop these, the x86_64 versions also don't look right if you compare them to the DataImportCrons HCO provides SSP:

```yaml
$ yq .[].metadata assets/dataImportCronTemplates/dataImportCronTemplates.yaml 
annotations:
  cdi.kubevirt.io/storage.bind.immediate.requested: "true"
name: centos-stream8-image-cron
labels:
  instancetype.kubevirt.io/default-preference: centos.stream8
  instancetype.kubevirt.io/default-instancetype: u1.medium
annotations:
  cdi.kubevirt.io/storage.bind.immediate.requested: "true"
name: centos-stream9-image-cron
labels:
  instancetype.kubevirt.io/default-preference: centos.stream9
  instancetype.kubevirt.io/default-instancetype: u1.medium
annotations:
  cdi.kubevirt.io/storage.bind.immediate.requested: "true"
name: fedora-image-cron
labels:
  instancetype.kubevirt.io/default-preference: fedora
  instancetype.kubevirt.io/default-instancetype: u1.medium
annotations:
  cdi.kubevirt.io/storage.bind.immediate.requested: "true"
name: centos-7-image-cron
labels:
  instancetype.kubevirt.io/default-preference: centos.7
  instancetype.kubevirt.io/default-instancetype: u1.medium
```

Reply:
https://github.com/kubevirt/containerdisks/pull/93